DM-32373: Run Gen 3 single frame measurement on on validation_data_cfht

parejkoj opened the pull request.

parejkoj committed:
Remove unneded files from gen2

Remove long-deprecated a.net files!

parejkoj committed:
Update refcats for gen3 and better coverage

Add script for making gen3 refcat ingest files.
Add gaia-dr2 refcat and gen3 ingest file.
Add sdss and ps1 gen3 refcat ingest files.
Add missing refcat files to cover whole exposure.

parejkoj committed:
Add processing output paths to gitgnore

parejkoj committed:
Add gen3 processing scripts and configs

parejkoj committed:
Update readme to reflect gen3 changes

parejkoj committed:
Remove pre-processed gen2 data

gen2 is going away soon, and these files were produced with a pipelines version
from many years ago, so they are not relevant at all now.

SimonKrughoff reviewed (approved):
This seems reasonable.  I'll take your word the scripts run as expected.

parejkoj committed:
Add setup line to ups table

Since this repo now contains processing scripts in bin/, it's useful to be able
to `setup -r .`
